lib/magma/model.rb

```diff
@@ -22,6 +22,11 @@ def load_attributes(attributes = [])
           @parent = attribute.name if attribute.is_a?(Magma::ParentAttribute)
           attribute.magma_model = self
           self.attributes[attribute.name] = attribute
+
+          if !attribute.is_a?(Magma::Link) && attribute.attribute_name != attribute.column_name
```

Comment:
We won't define aliases for link attributes with different `attribute_name`/`column_name` because they'll have Sequel association methods instead.